sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py

```diff
@@ -392,7 +392,14 @@ def make_named_trajectory(self, trajectory):
 
         time_from_start = 0.0
 
-        for wp in trajectory:
+        current_waypoint = {
+            'joint_angles': [],
+            'interpolate_time': 0.001,
+            'pause_time': 0.0
+        }
+        waypoints = [current_waypoint] + trajectory
```

Comment:
For this case, would it make sense to add a comment saying:
"We create a first trajectory point corresponding to the current position to avoid moveit complaining about it"